HealthMonitoring.AcceptanceTests/Scenarios/Selenium/Details_page.cs

```diff
@@ -0,0 +1,23 @@
+﻿using LightBDD;
+
+namespace HealthMonitoring.AcceptanceTests.Scenarios.Selenium
+{
+    [FeatureDescription(
+        @"In order to view endpoint details
+        As User 
+        I want to open details page")]
+    public partial class Details_page
+    {
+        [Scenario]
+        public void Details_page_structure_test()
```

Comment:
The scenario name should be describing more the scenario which we are going to test.

By looking at the steps, I think you are testing something like: Navigating_to_endpoint_details_page()